packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -61,9 +56,15 @@ class ChartController extends DisposableController
     this.displayXLabels = true,
     this.displayYLabels = true,
     this.name,
+    List<int> sharedLabelsTimestamps,
```

Comment:
nit: rename to labelTimestamps or sharedLabelTimestamps

Reply:
Done.